packages/config/src/Config.ts

```diff
@@ -489,11 +489,7 @@ function getRootPackageJsonPath(projectRoot: string, exp: ExpoConfig): string {
   return packageJsonPath;
 }
 
-export function readConfigJson(
-  projectRoot: string,
-  skipValidation: boolean = false,
-  skipNativeValidation: boolean = false
-): ProjectConfig {
+export function readConfigJson(projectRoot: string, skipValidation: boolean = true): ProjectConfig {
```

Comment:
```suggestion
export function readConfigJson(projectRoot: string, { skipValidation = true }: { skipValidation?: boolean } = {}): ProjectConfig {
```
We should get rid of these positional boolean arguments.
```
readConfigJson(projectRoot, true);  // what does true mean here? Need to look at the definition.
readConfigJson(projectRoot, { skipValidation: true });  // obvious at a glance
```

Reply:
Also `skipValidation` sounds like doing validation is the default and you can optionally skip it, but the default now is `skipValidation = true`. Should it be `false` instead? Or rename the option to `requireExpoSDKVersion`?